.gitignore

```diff
@@ -21,5 +21,6 @@ junit*.xml
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
+.idea/material_*.xml
```

Comment:
this is a file added by the material theme ui plugin, just adding it to .gitignore so i dont have to keep manually ignoring it